frontend/app/src/people/utils/ConnectCard.tsx

```diff
@@ -95,6 +95,8 @@ export default function ConnectCard(props: ConnectCardProps) {
   const showAssignModal = () => setAssignModal(true);
 
   const qrString = person && person?.owner_pubkey ? makeConnectQR(person?.owner_pubkey) : '';
+  const ownerPubkey = person && person?.owner_pubkey ? `${person.owner_pubkey}` : ''
+  const routeHint = person && person?.route_hint ? `:${person.route_hint}` : ''
```

Comment:
hi @kevkevinpal  the ':' addition was actually handled here if route_hint is available